src/test/scala/com/databricks/spark/redshift/RedshiftSourceSuite.scala

```diff
@@ -284,6 +284,7 @@ class RedshiftSourceSuite
 
   test("DefaultSource serializes data as Avro, then sends Redshift COPY command") {
     val params = defaultParams ++ Map(
+      "preactions" -> "DELETE FROM %s",
```

Comment:
Aside: it might be nice to have a dedicated unit test for this. I know that this has been kinda shoehorned into the existing unit test here for `postactions`, so if I was doing this over I'd move that out, too.